portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/wysiwyg/ManagementToolbar.testcase

```diff
@@ -223,43 +223,43 @@ definition {
 	@description = "LPS-147851. Assert the order button can display the correct icons"
 	@priority = "4"
 	test CanDisplayCorrectOrderIcons {
-		task ("And clicks on the order button") {
+		task ("And When order dropdown menu is opened") {
```

Comment:
Notice that I'm trying to avoid specific selenium actions like click and favoring more behavior focused terms.